src/types/qwsinglepixelbufferv1.h

```diff
@@ -11,22 +11,12 @@ struct wlr_single_pixel_buffer_manager_v1;
 QW_BEGIN_NAMESPACE
 
 class QWDisplay;
-class QWSinglePixelBufferManagerV1Private;
-class QW_EXPORT QWSinglePixelBufferManagerV1 : public QObject, public QWObject
+class QW_EXPORT QWSinglePixelBufferManagerV1
 {
-    Q_OBJECT
-    QW_DECLARE_PRIVATE(QWSinglePixelBufferManagerV1)
 public:
```

Comment:
QWSinglePixelBufferManagerV1() = delete;
~QWSinglePixelBufferManagerV1() = delete;

Reply:
Do it.